clang/test/Analysis/Checkers/WebKit/memory-unsafe-cast.cpp

```diff
@@ -0,0 +1,176 @@
+// RUN: %clang_analyze_cc1 -analyzer-checker=alpha.webkit.MemoryUnsafeCastChecker -verify %s
+
+class Base { };
```

Comment:
Can we add tests for static_cast non-pointer types and make sure they don't emit warnings?

Reply:
Added [https://github.com/llvm/llvm-project/pull/114606/files#diff-d68b9e1db69db208301dc5b614999fe9d99e88bed7de81b45416423008a4fbb5R28](https://github.com/llvm/llvm-project/pull/114606/files#diff-d68b9e1db69db208301dc5b614999fe9d99e88bed7de81b45416423008a4fbb5R28)